rllib/agents/dqn/apex.py

```diff
@@ -130,7 +130,7 @@ def __call__(self, item):
                     self.learner_thread.weights_updated = False
                     self.weights = ray.put(
                         self.workers.local_worker().get_weights())
-                actor.set_weights.remote(self.weights)
+                actor.set_weights.remote(self.weights, _get_global_vars())
```

Comment:
Not setting the global vars is a bug that would break schedules... not sure the best way to ensure we don't accidentally forget to update it.

Reply:
Yeah, I would like to make schedules easier to use for any hyperparams (where it makes sense to schedule it, not just lr). So, I'll revisit this problem soon anyways.